src/main/webapp/resources/angular/category/categoryList/category_list_controller.js

```diff
@@ -0,0 +1,24 @@
+(function () {
+    'use strict';
+
+    angular
+        .module('crowdfundingApp')
+        .controller('CategoryListController', CategoryListController);
+
+    CategoryListController.$inject = ['CategoryUtils'];
+
+    function CategoryListController(CategoryUtils) {
+
+        var vm = this;
+
+        loadingCategories();
+
+        function loadingCategories() {
+            CategoryUtils.getAllCategories()
```

Comment:
Modules that load some date should be named `Service`. Also? if you use `resource` you don't even need to add callback as it returns array that will be automatically populated.